recipes/philosopher/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "5.1.0" %}
+
+package:
+  name: philosopher
+  version: {{ version }}
+
+source:
+  url: https://github.com/Nesvilab/philosopher/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7807c50e488eda104f40a19b2fd29e393ab0b035fdaefc737892d744aaac51e4
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('philosopher', max_pin="x.x") }}
+  skip: true  # [osx]  # https://github.com/Nesvilab/philosopher/issues/21#issuecomment-3523415552
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - make
+    - unzip
+  run:
+    - libcxx  # or appropriate C library for binary if needed
```

Comment:
That is a bit wired. I guess you could add `- {{ compiler('cxx') }}` to get this.

Reply:
I could, but then I get all the compiler stuff which is not needed, or?

Reply:
I meant adding this to the build-section. So you will only get the runtime dependencies of the compilers, which I think is what you want here, do you?